src/notebooks/R Mia Examples/Comparative-Metagenomics.qmd

```diff
@@ -0,0 +1,852 @@
+---
+title: "Fetch Analyses metadata for a Study"
+author:
+  - name: Sam Hillman
+    affiliation:
+      - id: mia
+        name: miaverse team
+toc: true
+number-sections: true
+categories: [R]
+execute: 
+  eval: true
+---
+
+::: {style="max-width:1200px"}
+![](./../_resources/mgnify_logo.png)
+
+# Comparative metagenomics
+# Normalization methods, alpha & beta diversity, and differentially abundant taxa
+
+In this notebook we aim to demonstrate how the MGnifyR tool can be used to fetch data and metadata of a MGnify metagenomic analyisis. Then we show how to generate diversity metrics for comparative metagenomics using taxonomic profiles. Finally, we use the `mikropml` package to detect differentially abundant taxa and to build a ML classification model. 
+
+It's important to note, we're using methods and protocols that align with the [mia](https://microbiome.github.io/mia/) package, which can be read about further in the [OMA](https://microbiome.github.io/OMA/docs/devel/) guides.
+
+[MGnifyR](http://github.com/beadyallen/mgnifyr) is a library that provides a set of tools for easily accessing and processing MGnify data in R, making queries to MGnify databases through the [MGnify API](https://www.ebi.ac.uk/metagenomics/api/v1/). The benefit of MGnifyR is that data can either be fetched in tsv format or be directly combined in a phyloseq object to run an analysis in a custom workflow.
```

Comment:
Is the reference to phyloseq outdated here (the notebook is about TreeSE instead)? Check throughout